docs/cppcx/platform-guid-value-class.md

```diff
@@ -24,12 +24,13 @@ Guid has the Equals(), GetHashCode(), and ToString() methods derived from the [P
 |------------|-----------------|
 |[Guid](#ctor)|Initializes a new instance of the Guid struct.|
 |[operator==](#operator-equality)|Equals operator.|
-|[operator!=](#operator-not-equal)|Not equals operator.|
+|[operator!=](#operator-inequality)|Not equals operator.|
+|[operator<](#operator-less)|Less than operator.|
 |[operator()](#operator-call)|Converts a Guid to a GUID.|
 
 ### Remarks
 
-For an example of how to generate a new Platform::Guid using the Windows function [CoCreateGuid](/windows/desktop/api/combaseapi/nf-combaseapi-cocreateguid), see [WinRT component: How to generate a GUID?](http://blogs.msdn.com/b/eternalcoding/archive/2013/03/25/winrt-component-how-to-generate-a-guid.aspx)
+For an example of how to generate a new Platform::Guid using the Windows function [CoCreateGuid](/windows/desktop/api/combaseapi/nf-combaseapi-cocreateguid), see [WinRT component: How to generate a GUID?](https://www.eternalcoding.com/?p=383)
```

Comment:
Consider also linking to https://docs.microsoft.com/en-us/uwp/api/windows.foundation.guidhelper, which was created for JS, but is useful anywhere. Also may be worth calling out prefer Platform::GUID compare over GuidHelper::Equals in CX.